apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoader.java

```diff
@@ -869,6 +871,19 @@ static boolean isBeneficiaryHistoryEqual(
         newBeneficiaryRecord.getMbiEffectiveDate(), oldBeneficiaryRecord.getMbiEffectiveDate())) {
       return false;
     }
+
+    if (!Objects.equals(
```

Comment:
This will instruct the pipeline to add a new history record if the xref id or switch has changed